zabbix-agent2/config.json

```diff
@@ -1,22 +1,16 @@
 {
-  "name": "zabbix-agent2",
-  "version": "1.6.0",
+  "name": "Zabbix Agent 2",
+  "version": "1.7.0",
   "slug": "zabbix-agent2",
-  "description": "Zabbix agent 2",
+  "description": "Zabbix Network Monitoring Agent 2",
   "url": "https://github.com/pschmitt/home-assistant-addons/tree/main/zabbix-agent2",
   "arch": [
-    "armhf",
-    "armv7",
     "aarch64",
-    "amd64",
-    "i386"
```

Comment:
why remove these? are they not available upstream anymore?

Reply:
Yup only these archs are available upstream:

- linux/amd64
- linux/arm64
- linux/s390x
- linux/ppc64le

Ref: https://hub.docker.com/r/zabbix/zabbix-agent2/tags?name=alpine-7.0-latest